share/scripts/inverse/potential_extrapolate.sh

```diff
@@ -90,9 +90,12 @@ intermediate="$(critical mktemp "${input}.onlyleft.XXXXX")"
 if [[ $pot_type = "non-bonded"  ]]; then
   do_external table extrapolate --function ${lfct:-exponential} --avgpoints $avg_points --region left "${input}" "${intermediate}"
   do_external table extrapolate --function ${rfct:-constant} --avgpoints 1 --region right "${intermediate}" "${output}"
-elif [[ $pot_type = "bond"  || $pot_type = "angle" || $pot_type = "dihedral" ]]; then
+elif [[ $pot_type = "bond"  || $pot_type = "angle" ]]; then
   do_external table extrapolate --function ${lfct:-linear} --avgpoints $avg_points --region left "${input}" "${intermediate}"
   do_external table extrapolate --function ${rfct:-linear} --avgpoints $avg_points --region right "${intermediate}" "${output}"
+elif [[ $pot_type = "dihedral" ]]; then
+  do_external table extrapolate --function ${lfct:-linear} --avgpoints $avg_points --region left "${input}" "${intermediate}"
```

Comment:
Why would you not use `periodic` on the left side as well?

Reply:
So the scheme I have here is to extrapolate the left side normally and then extrapolate the right side to meet leftmost value. I think this scheme should suffice in most cases. A more elaborate scheme would be to pre-extrapolate on both sides, save the value at the boundaries, and extrapolate both sides to their average. But that would require a restructuring of the whole script and possibly others.